README.md

```diff
@@ -444,6 +444,7 @@ API | Description | Auth | HTTPS | CORS |
 | [GeoApi](https://api.gouv.fr/api/geoapi.html) | French geographical data | No | Yes | Unknown |
 | [Geocod.io](https://www.geocod.io/) | Address geocoding / reverse geocoding in bulk | `apiKey` | Yes | Unknown |
 | [Geocode.xyz](https://geocode.xyz/) | Provides worldwide forward/reverse geocoding, batch geocoding and geoparsing | No | Yes | Unknown |
+| [Geocodify.com](https://geocodify.com/) | Enterprise-grade geocoding, geoparsing and autocomplete for addresses | `apiKey` | Yes | Yes |
```

Comment:
```suggestion
| [Geocodify.com](https://geocodify.com/) | geocoding, geoparsing and autocomplete for addresses | `apiKey` | Yes | Yes |
```

Reply:
thanks for looking at this @SuperSandro2000